src/modules/openReserveWindow.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
- [ ] For best practice reasons and since it is a common practice in real working environment, I suggest fixing this listing issue instead of disabling linter checks here. Kindly find a way to resolve this without obstructing linter checks :pray: